src/main/resources/static/admin/styles.css

```
@@ -0,0 +1,325 @@
+}
+
+* {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	margin: 0;
+	padding: 0;
+	min-height: 100%;
+	font-family: "IBM Plex Sans", "Pretendard", "Noto Sans KR", sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)